src/components/ChildFormFragment.tsx

```diff
@@ -0,0 +1,131 @@
+        fieldKey={[field.fieldKey, 'lastName']}
+        label="Last Name"
+        className="inline-block-half"
+        rules={[{ required: true, message: 'Please input your last name' }]}
+      >
+        <Input placeholder="Last Name" />
+      </Form.Item>
+
+      <Form.Item
+        name={[field.fieldKey, 'pronouns']}
+        fieldKey={[field.fieldKey, 'pronouns']}
+        label="Pronouns"
+      >
+        <Radio.Group>
+          <Radio className="radio-item" value={'He/Him'}>
+            He/Him
+          </Radio>
+          <Radio className="radio-item" value={'She/Her'}>
+            She/Her
+          </Radio>
+          <Radio className="radio-item" value={'They/Them'}>
+            They/Them
+          </Radio>
+        </Radio.Group>
+      </Form.Item>
+
+      <Form.Item
+        name={[field.name, 'dateOfBirth']}
+        fieldKey={[field.fieldKey, 'dateOfBirth']}
+        label="Date of Birth"
+        rules={[
+          {
+            required: true,
+            message: 'Please input your date of birth.',
+          },
+        ]}
+      >
+        <DatePicker />
+      </Form.Item>
+      <Form.Item
+        name={[field.name, 'school']}
+        fieldKey={[field.fieldKey, 'school']}
+        label="School"
+        className="inline-block-half"
+        rules={[{ required: true, message: 'Please input your childs school' }]}
+      >
+        <Input placeholder="First Name" />
+      </Form.Item>
+
+      <Form.Item
+        name={[field.name, 'schoolYear']}
+        fieldKey={[field.fieldKey, 'schoolYear']}
+        label="Grade"
+        className="inline-block-half"
+        rules={[
+          { required: true, message: 'Please input your childs current grade' },
+        ]}
+      >
+        <Input placeholder="Last Name" />
+      </Form.Item>
+
+      <Form.Item
+        name={[field.name, 'allergies']}
+        fieldKey={[field.fieldKey, 'allergies']}
+        label="Allergies (if applicable)"
+      >
+        <TextArea rows={3} placeholder="Allergies" />
+      </Form.Item>
+
+      <Form.Item
+        name={[field.name, 'diagnosis']}
+        fieldKey={[field.fieldKey, 'diagnosis']}
+        label="Diagnosis"
+      >
+        <TextArea rows={1} placeholder="Diagnosis" />
+      </Form.Item>
+
+      <Form.Item
+        name={[field.name, 'medications']}
+        fieldKey={[field.fieldKey, 'medications']}
+        label="Medication (if applicable)"
+      >
+        <TextArea rows={1} placeholder="Medication" />
+      </Form.Item>
+
+      <Form.Item
+        name={[field.name, 'notes']}
+        fieldKey={[field.fieldKey, 'notes']}
+        label="Other Notes"
+      >
+        <TextArea rows={3} placeholder="Other Notes" />
+      </Form.Item>
+
+      <Form.Item
+        name={[field.name, 'profilePicture']}
+        fieldKey={[field.fieldKey, 'profilePicture']}
+        label="Upload Profile Picture"
+      >
+        <Dragger multiple={false} beforeUpload={() => false}>
```

Comment:
```suggestion
        <Dragger 
          multiple={false} 
          beforeUpload={() => false}
        >
```

Reply:
I think prettier disagrees so I wont make this change just based on that, I do think it might look better that way though!